### ISSUE TITLE
Could you provide a reference `.nml` file?

### ISSUE THREAD

**Issue by LemingVan** [SiqiLiOcean/HYCOM2FVCOM#1]
Hi @SiqiLiOcean ,

I am new to fvcom, thanks to your sharing, the scripts run very smooth.

But when I try to run the  model basd on hycom nesting, the model is stuck.

To be frankly, I don't know what problem to solve, because the fvcom output literally nothing.

So I wonder if you could provide some fvcom files relate to HYCOM2FVCOM?

Here is mine. [fvcom files.zip](https://github.com/SiqiLiOcean/HYCOM2FVCOM/files/14190263/fvcom.files.zip)

**Comment by SiqiLiOcean** [comment]
Hi,

Please send your log and error information when you run FVCOM.

Regards,
Siqi

**Comment by LemingVan** [comment]
I did three sets of experiments. 

The first one, `NESTING_ON = F`, the model works fine. 
The second one, `NESTING_ON = T`, `NESTING_BLOCKSIZE = -1`, model stuck. 
The third one, `NESTING_ON = T`, `NESTING_BLOCKSIZE = 100`, got the message **NESTING GRID MATCH ERROR**, the velocity of the model shows anomaly, and soon Nan appears. 

I think the problem is caused by nesting of open boundaries and would like to receive your suggestions.

[log_files.zip](https://github.com/SiqiLiOcean/HYCOM2FVCOM/files/14210948/log_files.zip)

**Comment by SiqiLiOcean** [comment]
The third one (NESTING_ON = T, NESTING_BLOCKSIZE = 100) is ok. There is no
error. 'NESTING GRID MATCH ERROR' is the difference between the model
original grid and nesting grid.

For the NaN on velocity, you may check if there is any NaN value in your
nesting file. You may check variables, including u, v, temp, salinity,
zeta, ua, and va.

**Comment by LemingVan** [comment]
Hi @SiqiLiOcean,

Thank you for your help! I checked the nesting file, and there was no NaN in the variables. turns out I set the wrong weight (all zeros) in the nesting file, which caused the OBC forcing is fixed in the nesting layers.

I want to share a modified download script. Considering the unstable network, the script can check and re-download files that have not been downloaded.

[hycom2fvcom_download_hycom_plus.sh.zip](https://github.com/SiqiLiOcean/HYCOM2FVCOM/files/14232579/hycom2fvcom_download_hycom_plus.sh.zip)
